ravenframework/Samplers/DynamicEventTree.py

```diff
@@ -923,6 +924,10 @@ def _localInputAndChecksHybrid(self,xmlNode, paramInput):
         self.hybridStrategyToApply[child.attrib['type']]._readMoreXML(childCopy)
         # store the variables that represent the epistemic space
         self.epistemicVariables.update(dict.fromkeys(self.hybridStrategyToApply[child.attrib['type']].toBeSampled.keys(),{}))
+        for epVar in self.epistemicVariables:
+          if len(epVar.split(",")) > 1:
+            for el in epVar.split(","):
+              self.fullyCorrelatedEpistemicToVar[el.strip()] = epVar
```

Comment:
is there a description about how epistemic variables are listed by the users?

Reply:
Yes the epistemic variables are all the variables listed in the definition of the ```<HybridSampler>```   within the 
```DynamicEventTree``` Samplers. This is already described in the manual.

For example in

>    
      <DynamicEventTree name='DET'>
        <HybridSampler type="MonteCarlo">
          <samplerInit>
            <limit>2</limit>
          </samplerInit>
          <variable name="missionTime,464:6">
            <distribution>missionTimeDist</distribution>
          </variable>
        </HybridSampler>
        <variable name='endtime'>
          <distribution>endtimedist</distribution>
          <grid type='CDF' construction='custom'>0.1 0.3 0.99</grid>
        </variable>
        <variable name='endtime2'>
          <distribution>endtime2dist</distribution>
          <grid type='CDF' construction='custom'>0.11 0.5 0.99</grid>
        </variable>
      </DynamicEventTree>

the epistemic variables are:
- ```missionTime```, ```464:6``` (which are fully correlated)

the aleatory variables are instead:
- ```endtime ```
- ```endtime2 ```